Prepare modern access control docs for release (#15204)
PrairieLearn/PrairieLearn: docs/instructor-guide/index.md

If you've taught with PrairieLearn before, here is a quick checklist to get you back up to speed:

> 1. **Copy your previous course instance** by opening last term from Course instances, going to Instance settings, and clicking **Make a copy**.

Author: ```suggestion
1. **Copy your previous course instance** by opening last the previous course instance, navigating to "Instance settings", and clicking **Make a copy**.
```
